<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAuditTreemap</class>
 <widget class="QWidget" name="FormAuditTreemap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualisation en Treemap</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cette fenêtre offre une visualisation en Treemap des données d'un audit.&lt;/p&gt;&lt;p&gt;Un &lt;a href=&quot;http://fr.wikipedia.org/wiki/Treemap&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;treemap&lt;/span&gt;&lt;/a&gt; est une représentation de données hiérarchiques en espace limité.&lt;/p&gt;&lt;p&gt;Chaque rectangle représente un noeud de l'arbre associé à l'audit actif, sa taille au sein du groupe de couleur est proportionnelle à la valeur sélectionnée dans la zone &amp;quot;Groupements&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="9" column="0" colspan="2">
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Enregistrer l'image</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="labelSelectedNodeName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelNodeName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QComboBox" name="comboListAudits"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="labelNodeValue">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Groupements</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QRadioButton" name="radioButtonGroupByORED">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Créer un regroupement de couleur différente pour les crédits ouverts, réalisés, engagés et disponibles. Les valeurs négatives ne sont pas représentées.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Par ORED</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QRadioButton" name="radioButtonGroupByNode">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Créer un seul regroupement pour le noeud sélectionné.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Par noeud</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioButtonGroupByMode">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Créer un regroupement de couleur différente pour les dépenses et recettes de fonctionnement/d'investissement.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Par mode</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="radioButtonGroupByYear">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Créer un regroupement de couleur différente par année.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Par année</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QComboBox" name="comboBoxYear"/>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="comboBoxORED"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QComboBox" name="comboBoxMode"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="TreeMapWidget" name="treeMapWidget" native="true"/>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QPushButton" name="pushButtonGoToPid">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Remonter au noeud père</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeMapWidget</class>
   <extends>QWidget</extends>
   <header>treemapwidget.h</header>
   <container>1</container>
   <slots>
    <signal>clicked(QString,int,int,double)</signal>
    <signal>highlighted(QString,int,int,double)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
